blocks/hero/hero.css

```diff
@@ -12,28 +12,85 @@ main .hero {
   position: relative;
   padding: 32px;
   min-height: 300px;
+  display: flex;
+  max-width: var(--max-content-width);
+  margin: auto;
+}
+
+main .hero .left-box {
+  background: #024c99;
+  min-height: 400px;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  overflow: hidden;
+  position: relative;
+}
+
+main .hero .right-box {
+  flex: 1;
+  display: none;
+}
+
+main .hero .left-box .bg-circle-1 {
+  background: #4179b3;
+  display: block;
+  position: absolute;
+  width: 390px;
+  height: 390px;
+  right: -90px;
+  top: -210px;
+  border-radius: 195px;
+  opacity: .4;
+}
+
+main .hero .left-box .bg-circle-2 {
+  background: #4179b3;
+  display: block;
+  position: absolute;
+  width: 220px;
+  height: 220px;
+  left: -110px;
+  bottom: 40px;
+  border-radius: 110px;
+  opacity: .4;
 }
 
 main .hero h1 {
-  max-width: 1200px;
-  margin-left: auto;
+  color: white;
   margin-right: auto;
+  padding: 48px;
+  z-index: 1;
+}
+
+main .hero h2 {
   color: white;
+  padding: 0 24px 48px 48px !important;
```

Comment:
anytime you need to use `!important` that is a good indication that something has gone wrong.

in this case, think we can just remove the important and it will still work

Reply:
Good catch ! my bad copy/paste from orig site.